src/qz/common/TrayManager.java

```diff
@@ -477,7 +474,7 @@ public void setServer(final Server server, final AtomicBoolean running, final At
             displayInfoMessage("Server started on port(s) " + TrayManager.getPorts(server));
 
             if (!headless) {
-                aboutDialog.setServer(server);
+                aboutDialog.usePort(PrintSocketServer.SECURE_PORTS.get(securePortIndex.get()));
```

Comment:
I've noticed this causes `localhost` to be hard-code lower in the code.  Can we improve the link a bit to launch at the IP that the server is running on, and not assume localhost?  That way if the SSL cert was re-issued, it's testing the same domain name that the cert was issued for.

Reply:
I forget that it might not always be on localhost. Changed to use the sever URI, which I think resolves this.